src/components/UI/Button/Button.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { FunctionComponent, MouseEventHandler } from 'react';
+import classes from './Button.module.css';
+
+type ButtonProps = {
+  onClick?: MouseEventHandler
+}
+
+// ToDo: create disable mode (with appropriate style)
+const button: FunctionComponent<ButtonProps> = props => {
+  return (
+    <button className={classes.Button} onClick={props.onClick}>
+      {props.children}
```

Comment:
JS/TS hint: you can use destructuring for props in place:

```ts
const button: FC<ButtonProps> = ({ onClick, children }) => (
  <button>...</button>
);
```

Reply:
I'll try this)